RecoEgamma/EgammaTools/python/hgcalElectronIDValueMap_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import dEdX_weights
+
+# cfi from HGCalElectronIDValueMapProducer::fillDescriptions()
+from RecoEgamma.EgammaTools.hgcalElectronIDValueMap_cfi import *
+hgcalElectronIDValueMap.dEdXWeights = dEdX_weights
```

Comment:
If this is the only way hgcalElectronIDValueMap is supposed to be used,
a safer pattern is to make hgcalElectronIDValueMapDefault in the fillDescriptions,
and then here clone hgcalElectronIDValueMap with an assignment of dEdX_weights.

Reply:
I would hope that at some point these sorts of intrinsic detector parameters would come from a CondFormat, since they would probably be calibrated IIUC.

Reply:
this is a good plan. 
However, we are not there yet and my feeling is not for a while